tests/test_executor.py

```diff
@@ -876,12 +878,17 @@ def test__pg_get_metadata(self, server_version, expected_value):
         mock_postgres.get_tablespaces.assert_called_once()
         mock_postgres.get_configuration_files.assert_called_once()
         if mock_postgres.server_version < 170000:
-            mock_postgres.get_setting.assert_called_once_with("data_directory")
+            calls = [call("data_directory"), call("data_checksums")]
+            mock_postgres.get_setting.assert_has_calls(calls, any_order=False)
```

Comment:
```suggestion
            mock_postgres.get_setting.assert_has_calls(calls, any_order=False)
            assert mock_postgres.get_setting.call_count == 2
```

To be even more precise 😄